pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/AbstractMetadataStore.java

```diff
@@ -306,4 +315,17 @@ protected static String parent(String path) {
 
         return path.substring(0, idx);
     }
+
+    /**
+     * valid path in metadata store should be
+     * 1. not blank
+     * 2. starts with '/'
+     * 3. not ends with '/', except root path "/"
+     */
+    public static boolean isValidPath(String path) {
```

Comment:
Maybe we can leave this as package protected

```suggestion
   static boolean isValidPath(String path) {
```